guides/dev/package_managers/README.md

```diff
@@ -28,6 +28,41 @@ It is thus recommended to use of the following strategies:
 
 Both options are explained in more detail in the [Setting CKEditor Configuration](http://docs.ckeditor.com/#!/guide/dev_configuration) article.
 
+## npm
+
+This article assumes that you have **npm** already installed and added to your `PATH`. If this is not the case, please follow the [installation guide](https://docs.npmjs.com/getting-started/installing-node).
+
+### Usage
+
+In order to fetch the most recent build execute the following command:
+
+	npm install ckeditor
+
+By default CKEditor will be placed in the `node_modules/ckeditor` directory.
+
+### Adding CKEditor as Dependency Using the `package.json` File
```

Comment:
I agree with your point. What I actually had in mind is to reduce it down to sth like:

In order to fetch the most recent build execute the following command: `npm install ckeditor --save` (see [npm installing tutorial](https://docs.npmjs.com/getting-started/installing-npm-packages-locally#installing) for more details).

As pointed out we could also simplify Bower part by removing "Adding CKEditor as Dependency Using the bower.json File" section and creating a proper mention / link to Bower docs for further reading.